docs/redirectless.md

```diff
@@ -0,0 +1,98 @@
+# Redirectless support
+In PingFederate, you can configure mobile applications to authenticate through REST APIs as OAuth clients without needing to handle HTTP redirections. When authentication is complete, the applications receive an OAuth authorization code or access token, and possibly an OpenID Connect ID token.
+
+Single-page web applications can also use redirectless mode if administrators configure them in PingFederate as authentication applications.
+## PingFederate configuration
+1. Follow the instructions at [README.md](../README.md#pingfederate-configuration) to configure Authentication API in PingFederate.
```

Comment:
This configuration looks to be for earlier versions than 10.1 and the below instructions are for 10.1. I think we need a strategy on how to handle spelling out these configurations. The PF config also seems overly restrictive for what the widget can support

Reply:
Looks like @brucem-ping created https://github.com/pingidentity/pf-authn-js-widget/issues/31 for this issue.